x-pack/legacy/plugins/code/server/indexer/commit_indexer.ts

```diff
@@ -104,6 +104,7 @@ export class CommitIndexer extends AbstractIndexer {
       this.commits = (await this.gitOps.log(this.repoUri, HEAD, Number.MAX_SAFE_INTEGER)).map(c => {
         const [message, ...body] = c.message.split('\n');
         return {
+          repoUri: this.repoUri,
```

Comment:
good catch. We had it in the mapping here: https://github.com/elastic/kibana/blob/9ff6d86379822b0ece43c58a70da5924b264b6b7/x-pack/legacy/plugins/code/server/indexer/schema/commit.ts#L11-L13.

But may forgot to populate this field. Thanks.